planning/behavior_velocity_planner/src/scene_module/virtual_traffic_light/scene.cpp

```diff
@@ -244,14 +246,18 @@ VirtualTrafficLightModule::VirtualTrafficLightModule(
   logger_ = logger_.get_child((map_data_.instrument_type + "_" + map_data_.instrument_id).c_str());
 }
 
-bool VirtualTrafficLightModule::modifyPathVelocity(
-  autoware_auto_planning_msgs::msg::PathWithLaneId * path,
-  tier4_planning_msgs::msg::StopReason * stop_reason)
+bool VirtualTrafficLightModule::modifyPathVelocity(PathWithLaneId * path, StopReason * stop_reason)
 {
   // Initialize
   setInfrastructureCommand({});
-  *stop_reason = planning_utils::initializeStopReason(
-    tier4_planning_msgs::msg::StopReason::VIRTUAL_TRAFFIC_LIGHT);
+  *stop_reason = planning_utils::initializeStopReason(StopReason::VIRTUAL_TRAFFIC_LIGHT);
+
+  // if (command_.type == "intersection_coordination") {
+  //   velocity_factor->detail = "intersection_coordination";
+  // } else if (command_.type == "eva_beacon_system") {
+  //   velocity_factor->detail = "eva_beacon_system";
+  // }
+
```

Comment:
Could you explain the reason for this commentout?

Reply:
Forgot to move the old code. Fixed it in https://github.com/autowarefoundation/autoware.universe/pull/1985/commits/8811ffc424fe41d1b169976cc7460ac8be528dcf.